src/anemoi/models/layers/mapper.py

```diff
@@ -518,6 +536,7 @@ def __init__(
         activation: str = "SiLU",
         mlp_extra_layers: int = 0,
         sub_graph: Optional[HeteroData] = None,
+        sub_graph_edge_attributes: Optional[list[str]] = None,
```

Comment:
This re-introduces hard graph assumptions for non-graph components to define names of attributes globally.